public/docs/ts/latest/guide/architecture.jade

```diff
@@ -53,14 +53,14 @@ figure
   Angular apps are modular and Angular has its own modularity system called _Angular Modules_ or _NgModules_.
```

Comment:
(I think. In general, Angular tends to capitalize things overmuch. Literal classnames should of course be capitalized, but I'm inclined not to capitalize other things like _dependency injection_ and _Angular modules_.)